README.md

```diff
@@ -9,15 +9,29 @@ These sample scanning applications demonstrate the capability of the [Genius Sca
 
 >Request a demo from [our website](https://geniusscansdk.com) to obtain a package containing the native iOS SDK (Objective-C libraries).
 
-# Content
-
-- [Android Simple demo](./android), which shows how to add a document scanner to your app in one line of code
-- [Android Custom demo](./android), which shows how to build a fully customized document scanner
-- [iOS Simple demo](./ios/GSSDKSimpleDemo), which shows how to add a document scanner to your app in one line of code
-- [iOS Custom demo](./ios/GSSDKCustomDemo), which shows how to build a fully customized document scanner
-- [React Native demo](./react-native-genius-scan-demo/README.md), which uses the [Genius Scan React Native component](https://www.npmjs.com/package/@thegrizzlylabs/react-native-genius-scan)
-- [Cordova demo](./cordova-plugin-genius-scan-demo/README.md), which uses the [Genius Scan SDK Cordova plugin](https://www.npmjs.com/package/@thegrizzlylabs/cordova-plugin-genius-scan)
-- [Flutter demo](./flutter-plugin-genius-scan-demo/README.md), which uses the [Genius Scan SDK Flutter plugin](https://pub.dev/packages/flutter_genius_scan)
+# Sample code
+
+- Android
```

Comment:
Let's be more specific and point to the `demo-simple` and `demo-custom` folders.